Make.inc

```diff
@@ -132,6 +132,10 @@ else
 JULIA_COMMIT := $(JULIA_VERSION)
 endif
 
+# Increase these every release when breaking ABI in a backward-incompatible way
```

Comment:
I think you can perfectly do that, that's even the solution required by Fedora when upstream does not set a SONAME: use SOMAJOR=0.X until upstream ships a release with SOMAJOR = 1. See https://fedoraproject.org/wiki/Packaging:Guidelines?rd=Packaging/Guidelines#Downstream_.so_name_versioning